tools/flang1/flang1exe/listing.c

```diff
@@ -116,8 +116,13 @@ list_ln(char *beg, char *txt)
     if (!lf)
       return; /* in case error msg written before file
                * opened */
+#ifdef FLANG_VENDOR
+    fprintf(lf, "\n\n\n%s%s (Version %8s)          %s      page %d\n\n",
+            FLANG_VENDOR, version.lang, version.vsn, gbl.datetime, pgno);
```

Comment:
Actually, instead of using the macro everywhere, perhaps the `version` struct should have a `vendor` field. Or we could repurpose the `product` field. What do you think?

Reply:
The main reason this macro was introduced is due to our LLVM guys demand to have flang's counterpart to their `CLANG_VENDOR`. I have no opinion on that, I've only materialized their will.